pypeit/scripts/ql_keck_mosfire.py

```diff
@@ -341,33 +397,50 @@ def main(args):
     # Create the pseudo images
     pseudo_dict = coadd.create_pseudo_image(coadd_dict_list)
 
-    ##########################
+    if args.flux:
+        # Load the sensitivity function
+        wave_sens, sfunc, _, _, _ = sensfunc.SensFunc.load(sensfuncfile)
```

Comment:
Does the archived sensitivity function cover the full wavelength of MOSFIRE?
If not, what happens?

Reply:
Still working on that. Let's clear this version first.

Reply:
What is the resolution?  A TODO?